Source/msg.h

```diff
@@ -198,6 +198,10 @@ enum _cmd_id : uint8_t {
 	// Cheat: increase active spell level of player.
 	//
 	// body (TCmd)
+	CMD_CHEAT_STATS,
+	// Cheat: max base stats of player.
+	//
+	// body (TCmd)

```

Comment:
If you remove the network message, you don't need to worry about this, but the comment should be above the corresponding enum value.